tests/test_lib_caconnector.py

```diff
@@ -299,3 +319,12 @@ def test_01_create_ca(self):
             self.assertEqual(cacon.workingdir, workdir)
             # check if the generated files exist
             self.assertTrue(os.path.exists(os.path.join(workdir, 'cacert.pem')))
+
+    def test_02_cleanup(self):
+        filelist = glob.glob("{0!s}2/*".format(WORKINGDIR))
+        for f in filelist:
+            try:
+                os.remove(f)
+            except:
+                print("Error deleting file {0!s}.".format(f))
```

Comment:
catching `IOError` here should be enough. Also we could issue a [warning](https://docs.pytest.org/en/latest/warnings.html) which pytest shows in the summary.

Reply:
will do

Reply:
done in 151a9d2d